services/worker-manager/src/providers/aws.js

```diff
@@ -260,13 +260,8 @@ class AwsProvider extends Provider {
   /**
    * This method checks instance identity document authenticity
    * If it's authentic it checks whether the data in it corresponds to the worker
-   *
-   * @param worker string
-   * @param workerPool string
-   * @param workerIdentityProof {document: string, signature: string}
-   * @returns {Promise<{expires: *}>}
    */
-  async registerWorker({worker, workerPool, workerIdentityProof}) {
+  async registerWorker({worker, workerPool, workerIdentityProof, secret}) {
```

Comment:
For future readers it might help to call this encryptedSecret, so we can see that it's already encrypted.